packages/extension-ui/src/Popup/Accounts/index.tsx

```diff
@@ -53,6 +53,7 @@ function Accounts({ className }: Props): React.ReactElement {
   }, []);
 
   const accounts = Object.entries(groupedParents)
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     .filter(([_, details]) => details.length > 0)
```

Comment:
You don't need that eslint-ignore - just ditch the unused variable (unlike for function params, it's possible for array destructuring):
```suggestion
    .filter(([, details]) => details.length > 0) 
```